packages/forms/resources/views/components/radio.blade.php

```diff
@@ -26,7 +26,7 @@
                 :attributes="$attributes->merge($getExtraAttributes())->class([
                     'filament-forms-radio-component',
                     'flex flex-wrap gap-3' => $isInline(),
-                    'gap-2' => ! $isInline(),
+                    'space-y-2' => ! $isInline(),
```

Comment:
What about using `flex flex-col`?

Reply:
Why introduce 2 classes if this can be solved by changing one?

Reply:
`flex` can be moved to outside the conditionals as it's also used for inline. Makes it easier to maintain if we're using flex and gap instead of margins in half of the places.

Reply:
Should we increase the gap to `gap-3`? Then we could move that outside the conditional, too.

Reply:
Not sure how that would be visually. Could you share a screenshot comparison?

Reply:
Gap could also be moved outside conditional using `gap-x-3` and `gap-y-2`.

Reply:
Maybe `flex-wrap` could also even be moved outside.